client/src/pages/Login.vue

```diff
@@ -13,11 +13,11 @@
             square
             ref="username"
             v-model="form.username"
-            :label="$t('auth.login_fields.username')"
+            :label="$t('auth.fields.username')"
             @keypress.enter="$refs.password.focus()"
             autofocus
           >
-            <template v-slot:prepend>
+            <template #prepend>
```

Comment:
Nothing that this is the only place in this file that uses the `v-slot` shorthand.

Reply:
I'll fix the other one.  Admittedly I don't think I intended to include this change, but I do prefer the shorthand (which is probably why it got picked up... I probably did a find and replace on s/v-slot:/#/ )

Reply:
Yeah, I noticed you changed it to the shorthand elsewhere. Seems like making this stylistic change consistent throughout the codebase could be a separate pull request itself.